presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1925,7 +1926,12 @@ public SystemSessionProperties(
                         featuresConfig.getDefaultViewSecurityMode(),
                         false,
                         value -> CreateView.Security.valueOf(((String) value).toUpperCase()),
-                        CreateView.Security::name));
+                        CreateView.Security::name),
+                booleanProperty(
+                        JOIN_PREFILTER_ENABLED,
```

Comment:
Since the prefilter is on build side, maybe the session property to reflect this?

Reply:
Done